CIP-????/README.md

```diff
@@ -0,0 +1,65 @@
+---
+CIP: ?
+Title: Integration of `keccak256` into Plutus
+Status: Proposed
+Category: Plutus
+Authors: 
+  - Sergei Patrikeev <so.patrikeev@gmail.com>
+  - Iñigo Querejeta-Azurmendi <inigo.querejeta@iohk.io>
+Implementors: []
+Discussions: 
+  - https://github.com/cardano-foundation/CIPs/pull/524
+Created: 2023-06-13
+License: Apache-2.0
+---
+
+## Abstract
+This CIP proposes an extension of the current Plutus functions to provide support for the [`keccak256`](https://keccak.team/files/Keccak-submission-3.pdf) hashing algorithm,
+primarily to ensure compatibility with Ethereum's cryptographic infrastructure.
+
+## Motivation: why is this CIP necessary?
+
+The [integration](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0049/README.md) of the ECDSA and Schnorr signatures over the secp256k1 curve into Plutus was a significant 
+step towards interoperability with Ethereum and Bitcoin ecosystems. However, full compatibility is still impossible 
+due to the absence of the `keccak256` hashing algorithm in Plutus interpreter, 
+which is a fundamental component of Ethereum's cryptographic framework:
+- data signing standard [EIP-712](https://eips.ethereum.org/EIPS/eip-712), 
+- `keccak256` is the hashing algorithm underlying Ethereum's ECDSA signatures. 
+- EVM heavily [depends](https://ethereum.github.io/yellowpaper/paper.pdf) on `keccak256` for internal state management
+
+Adding `keccak256` to Plutus would enhance the potential for cross-chain solutions between Cardano and EVM-based blockchains.
+
+## Specification
+This proposal aims to introduce a new built-in hash function `keccak_256`.
+
+This function will be developed following the [`keccak256`](https://keccak.team/files/Keccak-submission-3.pdf) specification 
+and will utilize the [cryptonite](https://github.com/haskell-crypto/cryptonite/blob/master/Crypto/Hash/Keccak.hs) implementation. 
```

Comment:
We do have that as a low priority task. The reason why we want to move away from it is because we have cryptonite as a dependency for a single primitive (SHA3), not due to any security concerns. However, we are not planning on moving away from cryptonite anytime soon, so using it's keccak implementation is the easiest. 

If we ever make this change in the near future (moving away from cryptonite) there will be no effects on the interface or behaviour of the primitives, so there would not need to be a hard-fork.

Reply:
noted... resolving these 2 `cryptonite` conversations to highlight practical blocks to merging the CIP.